packages/hadron-build/lib/download-center.js

```diff
@@ -1,198 +1,54 @@
+/* eslint-disable mocha/no-exports */
 const fs = require('fs-extra');
-const debug = require('debug')('mongodb-download-center');
-const AWS = require('aws-sdk');
+const { DownloadCenter } = require('@mongodb-js/dl-center');
 
+const DOWNLOADS_BUCKET = 'downloads.10gen.com';
 const MANIFEST_BUCKET = 'info-mongodb-com';
+const MANIFEST_OBJECT_KEY = 'com-download-center/compass.json';
 
-let maybeUpload = (CONFIG, asset) => {
-  const bucket = 'downloads.10gen.com';
-  const key = `${CONFIG.download_center_key_prefix}/${asset.name}`;
-
-  const params = {
-    Bucket: bucket,
-    Key: key,
-    Body: fs.createReadStream(asset.path),
-    ACL: 'public-read',
-    Metadata: {
-      Name: asset.name
-    }
-  };
-
-  return new Promise((resolve, reject) => {
-    new AWS.S3({
-      params: {
-        Bucket: bucket,
-        Key: key
-      }
-    }).headObject( (err, data) => {
-      if (data) {
-        debug(`Asset at s3://${bucket}/${key} already exists.  Skipping.`);
-        return resolve(data);
-      }
-
-      if (err && err.code !== 'NotFound') {
-        return reject(err);
-      }
-
-      const uploadReq = new AWS.S3.ManagedUpload({
-        params: params
-      });
-
-      uploadReq.send(function(uploadErr, res) {
-        if (uploadErr) {
-          debug(`uploading ${asset.name} failed: ${uploadErr}`);
-          return reject(err);
-        }
-        debug(`upload of ${asset.name} complete`);
-        resolve(res);
-      });
-
-      uploadReq.on('httpUploadProgress', (evt) => {
-        debug('got httpUploadProgress', evt);
-        if (!evt.total) return;
-        debug(`upload ${asset.name}: ${evt.total / evt.loaded}`);
-      });
-    });
-  });
-};
-
-let requireEnvironmentVariables = (keys) => {
+const requireEnvironmentVariables = (keys) => {
   for (let key of keys) {
     if (process.env[key]) return true;
     throw new TypeError(`Please set the environment variable ${key}`);
   }
 };
 
-let maybeUploadManifest = (CONFIG) => {
-  const prefix = `https://downloads.mongodb.com/${CONFIG.download_center_id}/${CONFIG.channel}`;
-  const bucket = 'info-mongodb-com';
-  const key = `com-download-center/${CONFIG.download_center_id}/${CONFIG.version}.json`;
-
-  const MANIFEST = {
```

Comment:
So this manifest would actually get uplaoded on every release, but it's not the one that is used for the download page and I am not really sure what it is in general: it's pretty outdated, download links are not correct and half of the supported OS is missing, so I just removed this. If anyone knows what this is used for, please tell!

Reply:
This thing was disabled at some point, not sure why, is safe to be removed

Reply:
Aha, good to know. Well, it's completely removed now